LibGetFrame-1.0.lua

```diff
@@ -590,25 +590,29 @@ function lib.GetUnitFrame(target, opt)
   if type(GetFramesCacheListener) ~= "table" then
     Init(true)
   end
-  opt = opt or {}
+  local defaultOpt
+  if not opt then
+	opt = {}
+	defaultOpt = true
+  end
   setmetatable(opt, { __index = defaultOptions })
 
   if not target then
     return
   end
 
   local ignoredFrames = CopyTable(opt.ignoreFrames)
-  if opt.ignorePlayerFrame then
+  if opt.ignorePlayerFrame and (not defaultOpt or target ~= "player") then
```

Comment:
imo the double negative makes it hard to read

(not a or not b) == not (a and b)

```Lua
if opt.ignorePlayerFrame and not (defaultOpt and target == "player") then
```